app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -258,6 +258,7 @@ class NavigationOverlayFragment : Fragment() {
         activity!!.moveTaskToBack(true)
     }
 
+    // TODO move this to the ToolbarUiController
```

Comment:
nit: were we supposed to address this or remove it? If not, can you elaborate in the comment on why should we move it (so other devs know why they might want to refactor this)?